common/libs/nodes/rpc/index.ts

```diff
@@ -16,6 +16,15 @@ import {
   isValidRawTxApi
 } from '../../validators';
 
+const timeout = (ms, promise: Promise<any>): any => {
+  return new Promise((resolve, reject) => {
+    setTimeout(() => {
+      reject(new Error('timeout'));
```

Comment:
Just in case there are some catches out there that display `error.message`, something like:
> The request timed out, try switching nodes or checking your connection